cfgov/jinja2/v1/owning-a-home/process/sources/index.html

```diff
@@ -0,0 +1,190 @@
+                            <p>The CFPB conducted an analysis using Equifax data provided by
+                                <a class="a-link
+                                          a-link__icon
+                                          cf-icon
+                                          cf-icon__after
+                                          cf-icon-external-link"
+                                    href="http://www.CreditForecast.com">
+                                    <span class="a-link_text">www.CreditForecast.com</span>
+                                </a>.  The analysis looked at mortgages taken out in 2006-2009.  More than half of the loans were closed within 5 years.</p>
+                        </div>
+                    </article>
+                </div>
+            </div>
+
+            <div class="block block__border-top block__padded-top block__flush-top block__flush-bottom">
+                <div class="content-l">
+                    <h2 class="content-l_col-1-4 content-l_col">
+                        <a href="/owning-a-home/process/close">Get ready to close</a>
+                    </h2>
+                    <article class="content-l_col-3-4 content-l_col">
+                        <div class="sources-step">
+                            <h3>Shop for title and other closing services</h3>
+                            <h4>&ldquo;Research suggests that borrowers who shop around for closing services could save as much as $500 on title services alone.&rdquo;</h4>
+                            <p>Feinberg, Robert, et al. (June 2012).  <i>What explains variation in Title Charges: A study of five large markets.</i> Prepared for the Department of Housing and Urban Development by The Urban Institute.  Available at:
+                                <a class="a-link
+                                          a-link__icon
+                                          cf-icon
+                                          cf-icon__after
+                                          cf-icon-external-link"
+                                    href="http://www.huduser.org/portal/publications/hsgfin/title_charges_2012.html">
+                                    <span class="a-link_text">
+                                        http://www.huduser.org/portal/publications/hsgfin/title_charges_2012.html
+                                    </span>
+                                </a>
+                            </p>
+                        </div>
+                    </article>
+                </div>
+            </div>
+
+            <div class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-top
+                        block__flush-bottom">
+                <div class="content-l">
+                    <h2 class="content-l_col-1-4 content-l_col">
+                        <a href="/owning-a-home/monthly-payment-worksheet">Monthly payment worksheet</a>
```

Comment:
Existing issue: This is 404ing -- apparently we didn't update the link when the monthly payment worksheet tool was taken down. It should go to the monthly payment worksheet pdf instead.